packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -1,29 +1,17 @@
 import { pxToRem } from '../../lib'
-import { colors, naturalColors } from './colors'
+import { colors } from './colors'
 
 //
 // COLORS
 //
-export { colors, contextualColors, emphasisColors, naturalColors, colorScheme } from './colors'
-
-export const gray02 = '#484644' // no mapping color
-export const gray03 = '#605E5C' // no mapping color
-export const gray04 = '#979593' // no mapping color
-export const gray06 = '#C8C6C4' // no mapping color
-export const gray08 = '#E1DFDD' // no mapping color
-export const gray09 = '#EDEBE9' // no mapping color
-export const gray10 = '#F3F2F1' // no mapping color
-export const gray14 = '#FAF9F8' // no mapping color
-
-export const brand04 = '#464775' // no mapping color
-export const brand08 = '#8B8CC7' // no mapping color
-
-export const magenta = '#B24782' // no mapping color
-export const orchid = '#943670' // no mapping color
-
-export const red = colors.red[900] // no mapping color $app-red
-export const red08 = '#F3D6DB' // no mapping color $app-red-08
-export const red10 = '#FCF4F6' // no mapping color $app-red-10
+export {
+  colors,
```

Comment:
do we really need to export all?
How do I know if I need to use `colors` vs `colorScheme` vs other type? As a user I'm confused by having so many choices

Reply:
We will have a guide for the colors usages. Basically, it should be something like this:
1. Whenever you need to use some color for a component, you should introduce variable for it, following the convention [aria][state], and if there is are props involved: [prop1][prop2][aria][state], for example: backgroundActive, or primaryBackgroundActive.
2. If your component supports color prop, then you should use color scheme, as you need to have all these areas + states defined per color
3. If you have a prop that somehow defines a color - then introduce a variable for it if it is not dependent on the color (like verticalBackgroundActive), or if it should be changed by altering colors, you may define two color schemes, default and vertical color scheme for example.
4. Last but not least :), you should have variables so that the user will in the end be able to change any of the areas + state. 

This is ideally how it should work. We cannot change all these in one go, currently we changed the Menu as one of the more complex components, as a showcase that this will work, but we need to take care of the implementation that we had before, and not introduce regressions. This is why currently in some case the color prop wins, in other case variable (this is how it previously behaved). As this evolves we will have one way of defining this. 

For the end user is important that they will have **one source of truth for colors** (one global color palette), and they will **always have to have at least variables for overriding the colors differently in all themes**. No color should be used directly in the styles.